mlibrary/pride
# `dependabot` group updates

# Overview
Dependabot was opening a pull request for each individual update. This pull request now groups `npm` updates and `github-actions` into their own pull requests. 

## Testing
1. Build the repository (`npm run build`).
2. Run the tests to make sure they pass (`npm run test`).
3. Navigate to your local [Search repository](https://github.com/mlibrary/search) and apply the newly generated files:
   1. Open `./package.json` and add `#dependabot-group-updates` at the end of the `pride` dependency URL:
      ```bash
      "pride": "git+https://github.com/mlibrary/pride.git#dependabot-group-updates"
      ``` 
   2. Do a clean install of Search:
      ```bash
      rm -rf node_modules && rm package-lock.json && npm install
      ``` 
   3. Start Search (`npm start`) and look around [the site](http://localhost:3000/everything). Check to see if there are any console errors, and everything still works as expected.
- _How to use the feature_.
- Make sure the PR is consistent in these browsers:
